<launch>
  <node pkg="laser_filters" type="scan_to_scan_filter_chain"
      name="laser_filters">
    <rosparam command="load" file="$(find mypkg)/PlatfrormFilter.yaml" />
    <remap from="scan" to="base_scan" />
  </node>
</launch>